examples/ropsten/contracts/Buffer.sol

```diff
@@ -1,132 +1,133 @@
 pragma solidity ^0.4.19;
 
 library Buffer {
-    struct buffer {
-        bytes buf;
-        uint capacity;
-    }
+  struct buffer {
+    bytes buf;
+    uint capacity;
+  }
 
-    function init(buffer memory buf, uint capacity) internal pure {
-        if(capacity % 32 != 0) capacity += 32 - (capacity % 32);
-        // Allocate space for the buffer data
-        buf.capacity = capacity;
-        assembly {
-            let ptr := mload(0x40)
-            mstore(buf, ptr)
-            mstore(ptr, 0)
-            mstore(0x40, add(ptr, capacity))
-        }
+  // solium-disable-next-line
```

Comment:
What's the justification for ignoring this?

Reply:
The error is

```
examples/ropsten/contracts/Buffer.sol
  9:2    error    "init": Avoid assigning to function parameters.    security/no-assign-params
```

The justification for ignoring it is:
1. It actually isn't our repo's code, it's being copied in from https://github.com/smartcontractkit/solidity-cborutils/blob/master/contracts/Buffer.sol#L9. Any change would need to be done there.
2. It assigns to the function parameter `buf` because it assigns it reserved memory on the heap. We could alter the function to return `buf` rather than modify a parameter, but because the change is in another repo, I think it's outside the scope of this story.

I do think it's a valid error. We can add a chore if you would like.

Reply:
I'd probably add a comment like: 

    // This code has been vendored in from ...url... so ignore warning, chore to fix here: https://pivotal...